packages/terra-base/docs/UPGRADEGUIDE.md

```diff
@@ -1,5 +1,14 @@
 # Terra Base Upgrade Guide
 
+## Changes from 4.0 to 5.0
+* To upgrade to terra-base v5, you now need to define a locale via the locale prop when using the `<Base>` component.
+* Any react-intl based exports that you were importing from terra-base should now be imported directly from react-intl.
+
+```diff
+- import { injectIntl, intlShape, FormattedMessage, IntlProvider } from terra-base;
++ import { injectIntl, intlShape, FormattedMessage, IntlProvider } from react-intl;
+```
```

Comment:
We need to add information regarding the removal of the post-install script and how this can impact testing localize components.  This is the main issue people are see issue with the post

Reply:
Ah, the post-install script was not a part of terra-base, that was a post-install script for terra-i18n. We have [documentation on how to test i18n based components here](https://engineering.cerner.com/terra-ui/#/getting-started/terra-ui/internationalization/testing-components-which-include-translations). I can add that to [terra-i18n upgrade guide](https://github.com/cerner/terra-core/blob/master/packages/terra-i18n/docs/UPGRADEGUIDE.md).

Reply:
Added more info on terra-i18n upgrade guide and referenced it from terra-base upgrade guide.
https://github.com/cerner/terra-core/pull/2459/commits/78fa28f00072e30883e45cecd261451c427f68a7